pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, date):
```

Comment:
Hi @MikeLing , could you please look at this code and make this PR look more like it?
https://github.com/AutomatedTester/Bugsy/blob/master/bugsy/bug.py#L17